resources/conf/config.toml

```diff
@@ -1,52 +1,57 @@
-[adapter]
 # The configuration file for mgw
-[adapter.server]
-host = "0.0.0.0"
-port = "9843"
 
-[[adapter.server.users]]
-username = "admin"
-password = "$env{adapter_admin_pwd}"
+[adapter] # --------------------------------------------------------
+
+# server: configuration related to REST API
+[adapter.server]
+  host = "0.0.0.0"
+  port = "9843"
+  tokenDuration = "1h"
```

Comment:
If we don't support `m` or `s` we shouldn't ask for that suffix. We should only expect a `int` value for the validity period in hours. We can add a comment saying `Token validity period in hours`.
Having `h` here is misleading and will cause users to try this with `m` and `s`

Reply:
1h is the default value. I not sure whether we decided not to support m.

Reply:
We might need to support m as well right? What we can do is, default unit it h, if not whatever the unit mentioned in the config.

Reply:
Adding a comment to the variable